app/resources/inventory.py

```diff
@@ -6,6 +6,16 @@
 from vars import game_info
 
 
+@m.user_endpoint(path=["inventory", "trade"], requires=trade_requirements)
+def trade(data: dict, user: str) -> dict:
+    element: Optional[Inventory] = wrapper.session.query(Inventory).filter_by(element_uuid=data["element_uuid"]).first()
+    if element is None or element.owner != user or element.owner == data["target"]:
+        return {"error": "item_not_found"}
+    element.owner = data["target"]
+    wrapper.session.commit()
+    return {"ok": True}
```

Comment:
use `success` from `schemes.py` instead
```suggestion
    return success
```